math/mathcore/test/CMakeLists.txt

```diff
@@ -5,9 +5,6 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 project(mathcore-tests)
-find_package(ROOT REQUIRED)
```

Comment:
Why here it was deleted while in `math/genetic/test/CMakeLists.txt ` if was organized as:
`if(NOT DEFINED ROOT_SOURCE_DIR)..endif()`

Reply:
I comment in `Mattermost` - you also need googletest and it looks very ugly. 
Therefore now it is not supported at all to build mathcore test standalone